src/custom-elements/src/components/text-input/text-input.tsx

```diff
@@ -138,7 +163,10 @@ export class TextInput {
       name={this.name || ''}
       disabled={this.disabled}
       id={this.el.id ? `${this.el.id}-control` : null}
-      onKeyUp={(ev) => this._handleValueChange(ev)}
+      onFocus={() => this.eventFocus.emit()}
+      onBlur={() => this.eventBlur.emit()}
+      onKeyUp={(ev) => this._handleValueInput(ev)}
```

Comment:
@janibolkvadze I thought you could to use onInput instead of onKeyUp. I guess you have a reason so, can you explain so in can know why? Thank you!

Reply:
If the reason is to support <IE9 versions I think it does not worth it. And using onInput you'll be able to listen for changes done by clicking on buttons (imagine input of type=number)